### ISSUE TITLE
How can I configure triggers? 

### ISSUE THREAD

**Issue by S0c5** [skymethod/denoflare#46]
first of all thanks for such amazing project, but I don't find a reference to see how can I configure my worker to use triggers.

**Comment by johnspurlock-skymethod** [comment]
Do you mean [cron triggers](https://developers.cloudflare.com/workers/platform/triggers/cron-triggers/)?

**Comment by S0c5** [comment]
Yes, it [includes triggers](https://developers.cloudflare.com/workers/wrangler/configuration/#triggers)!

**Comment by tbmale** [comment]
"Me too" :)
I would like to be able to specify cron triggers in .denoflare file like:
```json
"triggers":{
  "cron":"* * * * *"
}
```

**Comment by johnspurlock-skymethod** [comment]
This would be nice...
